### PR TITLE
doc: document control plane installation on GCP compute engine, closes HYB-363

### PR DESCRIPTION
**Pull request by tpetillot**

### CODE REVIEW

File: content/reference/admin/private_locations/installation/ecs/index.md [hunk 1]
```diff
@@ -211,4 +211,4 @@ After a short time, you should see your Control Plane get the "up" status in Gat
 
 {{< img src="ecs-control-plane-status.png" alt="Checking out the Control Plane's status in Gatling Enterprise Cloud" >}}
 
-You can now configure a simulation to run one one or more of this Control Plane's pool!
+You can now configure a simulation to run one one or more of this Control Plane's location!
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/ecs/index.md:214]:
```suggestion
You can now configure a simulation to run on one or more of this Control Plane's locations!
```

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+{{< /alert >}}
+
+In the GCP management console, open the [Secret Manager](https://console.cloud.google.com/security/secret-manager) (or search for "Secret Manager" in the search bar). 
+Set the right project and click on "Create Secret".
+
+Choose a name for the secret, and upload the [Control Plane configuration]({{< ref "../../configuration/gcp" >}})
+
+Configure other options as preferred.
+
+Click Create secret.
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:37]:
```suggestion
Click **CREATE SECRET**.
```

Comment (stb13579) [reply content/reference/admin/private_locations/installation/gcp/index.md:37]: Match the capitalization on the button and bold to indicate a button.

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+{{< img src="secret-configuration.png" alt="Configuring the secret" >}}
+
+## Service account
+
+We need a service account which will allow a VM to:
+
+- download the Control Plane's configuration file stored a secret
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:45]:
```suggestion
- download the Control Plane's configuration file, stored in a secret
```

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+- spawn new load generators on GCP when running a simulation
+
+### Role creation
+
+In the GCP management console, open [Roles](https://console.cloud.google.com/iam-admin/roles) (or search for "Roles" in the search bar).
+
+Required permissions:
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:52]: Does the user need to enable these permissions? If so: "Enable the following required permissions:"

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+```
+
+{{< img src="role-creation.png" alt="Role creation" >}}
+
+## Service account creation
+
+In the GCP management console, open [Service accounts](https://console.cloud.google.com/iam-admin/serviceaccounts) (or search for "Service Accounts" in the search bar). 
+Click on "Create service account".
+
+Fill details with a name, an ID and a description for the control plane service account.
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:71]:
```suggestion
Fill details with:
- a name, 
- an ID, 
- a description of the control plane service account.
```

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+{{< img src="service-account-details.png" alt="Service account details" >}}
+
+Then, choose the role previously created.
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:75]:
```suggestion
Then, choose the previously created role.
```

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+{{< img src="service-account-role.png" alt="Service account role" >}}
+
+Click on "Done".
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:79]:
```suggestion
Click "Done".
```

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+
+## VM instance creation
+
+In the GCP management console, open [VM Instances](https://console.cloud.google.com/compute/instances) (or search for "VM instances" in the search bar).
+Click on "Create instance".
+
+Set the name, region and zone of the VM instance.
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:86]:
```suggestion
Set the VM instance:
- name, 
- region, 
- zone.
```

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+
+Choose a machine type.
+
+{{< alert warning >}}
+Machine type with less than 1 shared core and 2GB of memory may take up to 10 minutes to deploy properly the instance.
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:91]:
```suggestion
A machine type with less than 1 shared core and 2GB of memory may take up to 10 minutes to deploy the instance properly.
```

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+_This is due to the startup agent [konlet](https://github.com/GoogleCloudPlatform/konlet) deploying container on VM, and toolbox gcloud CLI installation._
+{{< /alert >}}
+
+Container, click on "Deploy container".
+
+Set the image to `gatlingcorp/control-plane`, and then click on "Add a volume mount".
+
+Volume type is `Directory`, mount path is set to `/app/conf/control-plane.conf` and host path to `/etc/control-plane/control-plane.conf`.
+
+Control plane configuration from secret manager will be download to host path further on.
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:101]:
```suggestion
The control plane configuration from the secret manager will be downloaded to the host path later.
```

Comment (stb13579) [reply content/reference/admin/private_locations/installation/gcp/index.md:101]: I don't completely understand this, so make sure my change makes sense!

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+
+{{< img src="container-configuration.png" alt="Container configuration" >}}
+
+Configure the Identity and API access to the previously created service account.
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:105]:
```suggestion
Configure the Identity and API access for the previously created service account.
```

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+
+{{< img src="vm-identity-api-access.png" alt="Identity API Access" >}}
+
+Last step is to configure an automation that download the control plane configuration on start.
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:109]:
```suggestion
The last step is configuring an automation that downloads the control plane configuration on start.
```

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+
+To do so, go to advanced options, on Management section set Automation startup script to:
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:111]:
```suggestion
To do so, go to advanced options in the Management section and set the Automation startup script to:
```

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 14]
```diff
@@ -0,0 +1,134 @@
...
+sudo mkdir /etc/control-plane
+sudo touch /etc/control-plane/control-plane.conf
+
+toolbox gcloud version
+toolbox gcloud secrets versions access latest --secret=cp-config-file | sudo tee /etc/control-plane/control-plane.conf
+```
+
+{{< img src="vm-startup-script.png" alt="Identity API Access" >}}
+
+This script download the control plane configuration secret from secret manager on disk, on a path mounted on the control plane container.
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:124]:
```suggestion
This script downloads the control plane configuration secret from the secret manager on the disk to a path mounted on the control plane container.
```

Comment (stb13579) [reply content/reference/admin/private_locations/installation/gcp/index.md:124]: same comment, make sure I haven't changed the intended meaning

File: content/reference/admin/private_locations/installation/gcp/index.md [hunk 15]
```diff
@@ -0,0 +1,134 @@
...
+
+Click on "Create"
+
+## Your Control Plane is up and running!
+
+After a short time, you should see your Control Plane get the "up" status in Gatling Enterprise Cloud.
+
+{{< img src="gcp-control-plane-status.png" alt="Checking out the Control Plane's status in Gatling Enterprise Cloud" >}}
+
+You can now configure a simulation to run one one or more of this Control Plane's location!
```
Comment (stb13579) [inline content/reference/admin/private_locations/installation/gcp/index.md:134]:
```suggestion
You can now configure a simulation to run on one or more of this Control Plane's locations!
```

**Review by stb13579 — CHANGES_REQUESTED** [review]
Nice! Just a few minor English improvements and some formatting. In general, lists are better as bullet points for readability. Let me know if you need any clarification on my comments or suggestions.

**Comment by tpetillot** [comment]
Thanks a lot @stb13579 !

**Review by Obero — APPROVED** [review]
I agree with your doc, we just need to test it against reality with @bastien-gatling, who did not do a full set up like us.

**Review by bastien-gatling — APPROVED** [review]
Cool, juste a little comment !

**Pull request merged**